django_afip/models.py

```diff
@@ -543,6 +557,161 @@ def fetch_points_of_sales(
 
         return results
 
+    def get_caea(
```

Comment:
Yes I thought the same when I was naming it but then I thought that the `Taxpayer` "GETs" the Caea from AFIP and AFIP is who creates it.  It can be changed without problem

Reply:
You're right. Actually, I think `request_new_caea` makes most sense here, since that's what we do (and "Solicitar" is literally "request" too).